index.html

```diff
@@ -5,8 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href = "styles/index.css" rel="stylesheet" type="text/css">
 </head>
 <body>
-    
+    <section id ="headers">
+         <div id ="weather-header"><span class='bigger'>Weather Report</span> &nbsp;&nbsp; 
```

Comment:
Generally we try to avoid using something like `bigger` as a class name.  That way if instead of making the text bigger we decide to make it bold we don't have to rename everything:

```suggestion
         <div id ="weather-header"><span class='title'>Weather Report</span> &nbsp;&nbsp; 
```